src/services/coursesService.js

```diff
@@ -1,16 +1,27 @@
-const courses = require('../databases/coursesDb')
+const courses = require('../databases/coursesDb');
 
-const getCourses = ({page}) => {
-  return courses.getCourses({page, limit});
-}
+const getCourse = async ({ id }) => courses.getCourse({ id });
 
-const addCourse = ({description, name}) => {
+const getCourses = async ({ page, limit, userToken }) => {
+  // TODO: users service integration for the moment we use the token
+  const userId = userToken;
```

Comment:
jeje, buen intento.  Try to add a cliente like this https://github.com/math-learning/exercises-service/blob/chores/zube-62-exercise-service/src/clients/usersClient.js, that gets the metadata of the user.

I wll do the same this week, so you could just wait for me